CardData.py

```diff
@@ -77,9 +77,12 @@
     "Witcher": "Witcher"
 }
 
+
 class CardData:
-    def __init__(self, gwentDataHelper):
-        self._helper = gwentDataHelper
+    def __init__(self, gwent_data_helper):
+        helper = gwent_data_helper
+        self.patch = None
+        self.imageUrl = None
         self.cardTemplates = self._helper.getCardTemplates()
```

Comment:
broken, also avoid renaming the var to `helper`

Reply:
``self._helper`` was an object variable that is only used inside the constructor. That's why I made it a regular local variable.

Reply:
right, delete it and use `gwent_data_helper` directly